### PR TITLE
added cask for dymo-stamps

### PR DESCRIPTION
**Pull request by liljenstolpe**

### CODE REVIEW

File: Casks/dymo-stamps.rb [hunk 1]
```diff
@@ -0,0 +1,13 @@
+cask 'dymo-stamps' do
+  version :latest  # Upstream application upgraded in place
```
Comment (adityadalal924) [inline Casks/dymo-stamps.rb]: No need for this comment.

File: Casks/dymo-stamps.rb [hunk 2]
```diff
@@ -0,0 +1,13 @@
...
+  sha256 :no_check
+
+  url "http://download.endicia.com/dymostamps/dymostamps.dmg"
```
Comment (adityadalal924) [inline Casks/dymo-stamps.rb]: Casks/dymo-stamps.rb:5:7: C: Prefer single-quoted strings when you don't need string interpolation or special symbols.

File: Casks/dymo-stamps.rb (hunk 1 above)
Comment (liljenstolpe) [inline Casks/dymo-stamps.rb]: I did that, as I was asked to by another reviewer on another cask.  Sorry.

**Comment by liljenstolpe** [comment]
Will clean-up and re-submit

**Pull request closed without merging**